### ISSUE TITLE
Policy binding for application is blank and unclickable

### ISSUE THREAD

**Issue by kurokawaikki** [goauthentik/authentik#6694]
**Describe the bug**
After updating to 2023.08, I want to adjust the application policy/group/user binding. However, it is not possible to adjust them. After clicking on group, it will become blank (see screen shot 1).
If I refresh the page and try again, it will become unclickable (see screen shot 2).

**To Reproduce**

1. Go to 'Application'
2. Click on 'Bind existing policy'
3. Scroll down to 'group'
4. Become blank

**Expected behavior**
Be able to select group to bind.

**Screenshots**
1.
![image](https://github.com/goauthentik/authentik/assets/48047554/d7c7226f-fc51-451c-baab-9584a6e37148)

2.
![image](https://github.com/goauthentik/authentik/assets/48047554/0c97d40a-a0d1-4cb6-8945-023fdc5f3ac6)

**Logs**
No error in docker side. But error in web browser console.
rocket-loader.min.js:1  A preload for 'https://my.domain/static/dist/poly.js?version=2023.8.1' is found, but is not used because the request credentials mode does not match. Consider taking a look at crossorigin attribute.

A preload for 'https://my.domain/static/dist/standalone/loading/index.js?version=2023.8.1' is found, but is not used because the request credentials mode does not match. Consider taking a look at crossorigin attribute.

A preload for 'https://my.domain/static/dist/admin/AdminInterface.js?version=2023.8.1' is found, but is not used because the request credentials mode does not match. Consider taking a look at crossorigin attribute.

Access to fetch at 'https://authentik.error-reporting.a7k.io/api/4504163677503489/envelope/?sentry_key=151ba72610234c4c97c5bcff4e1cffd8&sentry_version=7&sentry_client=sentry.javascript.browser%2F7.65.0' from origin 'https://my.domain' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

POST https://authentik.error-reporting.a7k.io/api/4504163677503489/envelope/?sentry_key=151ba72610234c4c97c5bcff4e1cffd8&sentry_version=7&sentry_client=sentry.javascript.browser%2F7.65.0 net::ERR_FAILED 503

PolicyBindingForm.ts:85 
 
 POST https://my.domain/api/v3/policies/bindings/ 400
Qe.t	@	instrument.js:142
(anonymous)	@	runtime.js:86
a	@	runtime.js:17
Promise.then (async)		
u	@	runtime.js:19
(anonymous)	@	runtime.js:20
i	@	runtime.js:16
fetchApi	@	runtime.js:77
(anonymous)	@	runtime.js:155
a	@	runtime.js:17
Promise.then (async)		
u	@	runtime.js:19
(anonymous)	@	runtime.js:20
i	@	runtime.js:16
request	@	runtime.js:153
(anonymous)	@	PoliciesApi.js:300
(anonymous)	@	PoliciesApi.js:20
ku	@	PoliciesApi.js:16
policiesBindingsCreateRaw	@	PoliciesApi.js:290
(anonymous)	@	PoliciesApi.js:315
(anonymous)	@	PoliciesApi.js:20
ku	@	PoliciesApi.js:16
policiesBindingsCreate	@	PoliciesApi.js:314
value	@	PolicyBindingForm.ts:85
(anonymous)	@	Form.ts:229
d	@	regeneratorRuntime.js:44
(anonymous)	@	regeneratorRuntime.js:125
(anonymous)	@	regeneratorRuntime.js:69
r	@	asyncToGenerator.js:3
s	@	asyncToGenerator.js:22
(anonymous)	@	asyncToGenerator.js:27
(anonymous)	@	asyncToGenerator.js:19
(anonymous)	@	Form.ts:289
(anonymous)	@	ModalForm.ts:30
d	@	regeneratorRuntime.js:44
(anonymous)	@	regeneratorRuntime.js:125
(anonymous)	@	regeneratorRuntime.js:69
r	@	asyncToGenerator.js:3
s	@	asyncToGenerator.js:22
(anonymous)	@	asyncToGenerator.js:27
(anonymous)	@	asyncToGenerator.js:19
(anonymous)	@	ModalForm.ts:54
(anonymous)	@	ModalForm.ts:86
task	@	BaseTaskButton.ts:66
run	@	task.js:6
value	@	BaseTaskButton.ts:101
handleEvent	@	lit-html.js:6
r	@	helpers.js:90
c	@	rocket-loader.min.js:1

Access to fetch at 'https://authentik.error-reporting.a7k.io/api/4504163677503489/envelope/?sentry_key=151ba72610234c4c97c5bcff4e1cffd8&sentry_version=7&sentry_client=sentry.javascript.browser%2F7.65.0' from origin 'https://my.domain' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

POST https://authentik.error-reporting.a7k.io/api/4504163677503489/envelope/?sentry_key=151ba72610234c4c97c5bcff4e1cffd8&sentry_version=7&sentry_client=sentry.javascript.browser%2F7.65.0 net::ERR_FAILED 503

**Version and Deployment (please complete the following information):**

-   2023.08
-   Docker-compose

**Comment by BeryJu** [comment]
duplicate of #6691
